[DOCS] Event filters and Host isolation exceptions by Policy (#1615)
elastic/security-docs: docs/management/admin/event-filters.asciidoc

Event filters allow admins to filter endpoint events that you do not need or want stored in {es} -- for example, those with high volumes. By creating event filters, you can optimize your storage in {es}. All endpoint events have the `endpoint.events.network` field.

> By default, an event filter is recognized globally across all hosts running {endpoint-sec}. If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], you can also assign an event filter to a specific {endpoint-sec} integration policy, filtering endpoint events from only the hosts assigned to that policy.

Reviewer: ```suggestion
By default, an event filter is recognized globally across all hosts running {endpoint-sec}. If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], you can also assign an event filter to a specific {endpoint-sec} integration policy, which would filter endpoint events from only the hosts assigned to that policy.
```

just putting it out there

> By default, an event filter is recognized globally across all hosts running {endpoint-sec}. If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], you can also assign an event filter to a specific {endpoint-sec} integration policy, which would filter endpoint events from only the hosts assigned to that policy.

Reviewer 2: ```suggestion
By default, an event filter is recognized globally across all hosts running {endpoint-sec}. If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], you can also assign an event filter to a specific {endpoint-sec} integration policy, which would filter endpoint events from the hosts assigned to that policy.
```

[role="screenshot"]
image::images/event-filter.png[]
. Enter a name for the event filter.

> . Enter a description that identifies the filter (optional).

Reviewer: ```suggestion
. Enter a filter description (optional).
```
Does it really identify (like an ID #?), or just describe?

> . Depending on which page you add the filter, either modify the pre-populated conditions or add new conditions that define when {elastic-sec} filters events. You can define multiple conditions with `AND` relationships. You can also add nested conditions. For example, in the image above, the event filter excludes events whose `event.category` field is `network`, and whose `process.executable` field is the same as the specified path.

Reviewer: ```suggestion
. Depending which page you are using to create the filter, either modify the pre-populated conditions or add new conditions to define when {elastic-sec} filters events. You can define multiple conditions with `AND` relationships. You can also add nested conditions. For example, the event filter pictured above excludes events whose `event.category` field is `network`, and whose `process.executable` field is as specified.
```

. Select an option in the *Assignment* section to assign the event filter to a specific integration policy:
+
* `Global`: Assign the event filter to all integration policies for {endpoint-sec}.
* `Per Policy` (Platinum or Enterprise subscription only): Assign the event filter to one or more specific {endpoint-sec} integration policies. Select each policy in which you want the events to be filtered.
+

> NOTE: You can also select the `Per Policy` option without immediately assigning a policy to the event filter. For example, you could do this to create and review your event filter configurations before putting them into action with a policy.

Reviewer 2: ```suggestion
NOTE: You can also select the `Per Policy` option without immediately assigning a policy to the event filter. For example, you could do this to create and review your event filter configurations before activating them with a policy.
```

Reviewer 2: Not sure if this is how it works, but offering the suggestion nevertheless in case it is.

Reviewer 2: Ah ok, I see what you're saying. In that case, maybe "enacting"? Also, the original works as is, so whatever you choose!

> . Add a new comment if you want to provide more information about the event filter (optional).

Reviewer: ```suggestion
. Add a comment if you want to provide more information about the event filter (optional).
```

. Click *Add event filter*. The new filter is added to the *Event filters* list.

[[manage-event-filters]]
[discrete]
== View and manage event filters

> The **Event filters** list allows you to view and manage all endpoint event filters that have been added. To view the Event filters list, go to *Manage* -> *Event filters*. Event filters appear in reverse chronological order, with the most recently created filter at the top. Each filter has its own entry, which displays details such as the filter name, operating system, date created, and the filter conditions.

Reviewer: ```suggestion
The **Event filters** list allows you to view and manage your endpoint event filters. To view the Event filters list, go to *Manage* -> *Event filters*. Event filters appear in reverse chronological order, with the most recently created at the top. Each filter has its own entry, which displays details such as the filter's name, operating system, date created, and conditions.
```

> To refine the **Event filters** list, enter a query in the search bar. You can search the by name, comments, or the value of a field.

Reviewer: ```suggestion
To refine the **Event filters** list, enter a query in the search bar. You can search filter names, comments, and field values.
```

[role="screenshot"]
image::images/event-filters-list.png[]

[discrete]
[[edit-event-filter]]
=== Edit an event filter

> You can change the configuration of each event filter. If you have a Platinum or Enterprise subscription, you can also change the policy assigned to an event filter.

Reviewer: ```suggestion
You can individually configure each event filter. With a Platinum or Enterprise subscription, you can also change the policy or policies assigned to each filter.
```

Author: Going with this + a slight tweak:
```suggestion
You can individually configure each event filter. With a Platinum or Enterprise subscription, you can also change the policies assigned to each filter.
```
